### ISSUE TITLE
Laut leiser doesn't work

### ISSUE THREAD

**Issue by Pecurse** [PRProd/HA-Firemote#605]
### Describe the bug

The buttons for Volume + and - doesn't work on fire tv stick 4K(2nd Gen) on layout "App Launcher 2"

### To Reproduce

![Screenshot_20241230_185433_io homeassistant companion android](https://github.com/user-attachments/assets/eeac2c46-7a67-4ef1-826d-f2ed18e7be2a)

### Expected Behavior

I tried it multiple times, but volume stay at the same

### Which Type of Device Do You Own?

Fire tv stick 4k (2nd Gen)

**Comment by PRProd** [comment]
Greetings @Pecurse!

Congratulations on opening your first issue!

Have you had a chance to take a look at the [Firemote FAQ](https://github.com/PRProd/HA-Firemote?tab=readme-ov-file#faq) section of the [README](https://github.com/PRProd/HA-Firemote/blob/main/README.md) file?  If not, I highly recommend looking at the 2nd and third questions / answers that are listed there:
 - [Why won't the volume, mute, and/or power buttons work from my Firemote card?](https://github.com/PRProd/HA-Firemote?tab=readme-ov-file#why-wont-the-volume-mute-andor-power-buttons-work-from-my-firemote-card)
 - [Why don't any of the buttons on the Firemote work at all?](https://github.com/PRProd/HA-Firemote?tab=readme-ov-file#why-dont-any-of-the-buttons-on-the-firemote-work-at-all)
 
I'm hoping that one of those answers will help you.  If not, please let me know.  Good luck!

-Doug

**Comment by PRProd** [comment]
@Pecurse ~

Have you checked the menus on your TV to see if CEC is enabled there too?

**Comment by PRProd** [comment]
@Pecurse ~ Any update here?  Were you able to check your TV?

**Comment by Pecurse** [comment]
Yeah, I've tried it an activated  the CEC, but still doesn't work.

Gesendet von Outlook für Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by PRProd** [comment]
@Pecurse ~

Sometimes an old or bad HDMI cable can cause CEC to not work properly.  This is also true if you are running the signal through another device first.

Are you using one single cable between your device and your TV?  Have you tried swapping it out with a different cable?

**Comment by slhrlsc** [comment]
Hello, Fire tv stick 4k (2nd Gen) here, on a Hisense TV.
Same case with Volume Up/Down and Mute buttons, they don't work. The other button do.
CEC turn on on the Firestick and TV as well.
No HDMI cable used, stick is connected directly to the port on TV.

**Comment by austonpramodh** [comment]
I am having same issue with Fire TV Cube 2nd Gen, I have disconnected IR blaster, and volume everything works through Alexa. Remote seems to be using IR Blaster, doesnt work when i cover.


Lemme know how can i help/

TV: Toshiba Fire TV

**Comment by duczz** [comment]
we need to set something like this, so it can be closed.
https://github.com/PRProd/HA-Firemote?tab=readme-ov-file#button-overrides

```
button_overrides:
  volume-up-button:
    action: media_player.volume_up
    target:
      device_id: aa5de480ded3
  volume-down-button:
    action: media_player.volume_down
    target:
      device_id: aa5de480ded3
```
